server/container/guice/guice-common/src/test/java/org/apache/james/PeriodicalHealthChecksTest.java

```diff
@@ -100,7 +101,7 @@ void startShouldCallHealthCheckAtLeastOnce() {
         testee.start();
 
         scheduler.advanceTimeBy(Duration.ofSeconds(PERIOD));
-        verify(mockHealthCheck1, atLeast(1)).check();
+        verify(mockHealthCheck1, atLeast(1)).checkReactive();
```

Comment:
does it work if you don't call block on checkReactive ?

Reply:
or you should use awaitility ?

Reply:
Sorry i don't understand your point here. I started the healthcheck and didn't call any block()? 

Or do you mean the actual execution that we expected checkReactive to do without calling block()? I tested that already in the following cases.

Reply:
ah ok we don't mind about the actual execution. i didn't understood.